src/ContainerDetails.jsx

```diff
@@ -74,19 +130,27 @@ const ContainerDetails = ({ container, containerDetail }) => {
                     <DescriptionListTerm>{_("Gateway")}</DescriptionListTerm>
                     <DescriptionListDescription>{containerDetail.NetworkSettings.Gateway}</DescriptionListDescription>
                 </DescriptionListGroup>}
-                {containerDetail && containerDetail.NetworkSettings.MacAddress && <DescriptionListGroup>
-                    <DescriptionListTerm>{_("MAC address")}</DescriptionListTerm>
-                    <DescriptionListDescription>{containerDetail.NetworkSettings.MacAddress}</DescriptionListDescription>
+                {ports && <DescriptionListGroup>
+                    <DescriptionListTerm>{_("Ports")}</DescriptionListTerm>
+                    <DescriptionListDescription>{ports}</DescriptionListDescription>
                 </DescriptionListGroup>}
             </DescriptionList>}
-            <DescriptionList className='container-details-state'>
+            <DescriptionList className='container-details-networks'>
                 <DescriptionListGroup>
-                    <DescriptionListTerm>{_("Created")}</DescriptionListTerm>
-                    <DescriptionListDescription>{utils.localize_time(Date.parse(container.Created) / 1000)}</DescriptionListDescription>
+                    <DescriptionListTerm>{_("Networks")}</DescriptionListTerm>
+                    <DescriptionListDescription>{networks}</DescriptionListDescription>
                 </DescriptionListGroup>
+            </DescriptionList>
+            <DescriptionList className='container-details-volumes'>
```

Comment:
Should not shown when volumes are empty.

Reply:
agreed